docs/docs-graphql/schema.graphqls

```diff
@@ -514,6 +514,29 @@ type Container {
     source: FileID!
   ): Container!
 
+  """
+  Retrieves this container plus the contents of the given files copied to the given path.
+  """
+  withFiles(
+    """Identifiers of the files to copy."""
+    files: [FileID!]!
```

Comment:
```suggestion
    """Identifiers of the files to copy."""
    sources: [FileID!]!
```

We should mirror the structure of `withFile`.
- `withFile` :arrow_right: `withFiles`
- `source` :arrow_right: `sources`